src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/RemoveApplicantDetailsControllerTest.java

```diff
@@ -36,8 +36,8 @@ public void shouldSuccessfullyRemoveApplicantSolicitorDetails() throws Exception
                 .andExpect(jsonPath("$.data.applicantAddress", is(notNullValue())))
                 .andExpect(jsonPath("$.data.applicantPhone", is("89897876765")))
                 .andExpect(jsonPath("$.data.applicantEmail", is("email01@email.com")))
-                .andExpect(jsonPath("$.data.applicantFMName", is("Test")))
-                .andExpect(jsonPath("$.data.applicantLName", is("Test")))
```

Comment:
This is the issue with having duplicated keys in the test JSON files - not clear which exactly we were testing